geoprocessing/src/modules/surface-cost/adapters/typeorm-cost-surface.ts

```diff
@@ -3,27 +3,24 @@ import { Injectable } from '@nestjs/common';
 import { Repository } from 'typeorm';
 import { flatten } from 'lodash';
 
-import { CostSurfaceRepo } from '../cost-surface-repo';
-import { CostSurfaceInputDto } from '../../../entry-points/adjust-cost-surface-input';
-import { ScenariosPuCostDataGeo } from './scenarios-pu-cost-data.geo.entity';
-import { DbConnections } from '../../../../../ormconfig.connections';
-
-type Success = true;
+import { CostSurfacePersistencePort } from '../ports/persistence/cost-surface-persistence.port';
+import { PlanningUnitCost } from '../ports/planning-unit-cost';
+import { ScenariosPuCostDataGeo } from '../../scenarios/scenarios-pu-cost-data.geo.entity';
 
 @Injectable()
-export class TypeormCostSurface implements CostSurfaceRepo {
+export class TypeormCostSurface implements CostSurfacePersistencePort {
```

Comment:
adapter only adjusts to a bit different port (and environment of geoprocessing service /db/)